tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java

```diff
@@ -93,7 +93,12 @@
      * The baseline CLDRFile for this DataSection
      */
     public CLDRFile translationHintsFile;
-    
+
+    /**
+     * The CLDRFile for the root locale; use lazy initialization, see getRootFile
+     */
+    private CLDRFile rootFile = null;
```

Comment:
Note for all.
There are three files that must never change while the ST is active:
en.xml
root.xml
and the translationHints file.

So we could have statics for those to save building them for any new instance.

Reply:
I'll make a ticket for that. Maybe the statics should be for all of ST, not just DataSection?